_locales/pt_BR/messages.json

```diff
@@ -1,11 +1,11 @@
 {
     "badgeTooltip": {
-        "message": "1 bitcoin vale $1 hoje",
-        "description": "Text that shows up when the mouse is hovering over the add-on's button in the browser's toolbar."
+        "message": "1 bitcoin vale US $1 hoje",
```

Comment:
This is not a currency symbol, it's a placeholder. Leave it as is.

Reply:
Updated 👍